src/components/project-header.tsx

```diff
@@ -1,26 +1,15 @@
 /** @jsx jsx */
-import { jsx, Link } from "theme-ui"
-import ProjectPicture from "./project-picture"
+import { jsx } from "theme-ui"
 import Status from "./status"
+import ProfileHeader from "./profile-header";
 
 const ProjectHeader = (props) => {
-
   return (
-    <div style={{display: "flex", flexDirection: "row", verticalAlign: `middle`}}>
-      <ProjectPicture src={props.image} slug={`${props.title.toLowerCase().split(" ").join("-")}`} />
-      <div sx={{ display: "flex", 
-              alignItems: "center", 
-              paddingLeft: "25px", 
-              paddingRight: "25px" 
-      }}>
-          <h2 sx={{ fontSize: [4, 5, 6], fontWeight: `bold`, color: `title` }}>
-              {props.title}
-          </h2>
-      </div>
+    <ProfileHeader image={props.image} title={props.title}>
```

Comment:
So I thought there were a lot of similarities between `ProfileHeader` and `ProjectHeader` so I thought why don't I just make `ProjectHeader` a `ProfileHeader` component with the added status component to prevent me from rewriting responsive code for both.